### PR TITLE
20 sign in

### PR DESCRIPTION
**Pull request by alaa-yasin**
In this branch
I created a sign in form with validation, use common components like the header and google and facebook buttons. 
Relates #20

### CODE REVIEW

**Comment by Fatmasiam** [comment]
where is the PR description ! and relates!

**Comment by Fatmasiam** [comment]
fix the page style

File: .storybook/config.js [hunk 1]
```diff
@@ -1,5 +1,13 @@
 import { configure } from '@storybook/react';
+<<<<<<< HEAD
```
Comment (Fatmasiam) [inline .storybook/config.js]: conflict!

File: src/containers/SignIn/index.js [hunk 2]
```diff
@@ -1,8 +1,110 @@
...
+  const {
+    form: { getFieldDecorator, validateFieldsAndScroll },
+    history: { goBack },
+  } = props;
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    validateFieldsAndScroll((err, values) => {
+      if (!err) {
+        // eslint-disable-next-line no-console
```
Comment (Fatmasiam) [inline src/containers/SignIn/index.js]: disable eslint !

File: src/containers/SignIn/index.js [hunk 3]
```diff
@@ -1,8 +1,110 @@
...
+              />
+            )}
+          </Form.Item>
+
+          <Form.Item hasFeedback>
+            {getFieldDecorator('password', {
+              rules: [
+                {
+                  pattern: new RegExp(
+                    /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d].{8,}$/
```
Comment (Fatmasiam) [inline src/containers/SignIn/index.js]: we don't need the regex here ?!! 
in the sign-in we just need it to be required

File: src/containers/SignIn/utils/signInSchema.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import * as yup from 'yup';
```
Comment (Fatmasiam) [inline src/containers/SignIn/utils/signInSchema.js]: i think we don't need this file

File: src/containers/SignIn/signIn.css [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+  }
+  .signin svg {
+    width: 25px;
+    height: 25px;
+    margin-right: 20px
+  }
+  .ant-input-affix-wrapper .ant-input:not(:first-child) {
+    padding-left: 50px !important;
+  }
+}
```
Comment (Fatmasiam) [inline src/containers/SignIn/signIn.css]: add new line

File: src/containers/SignIn/index.js [hunk 6]
```diff
@@ -1,8 +1,105 @@
...
+
+import Header from '../../components/Header';
+import FacebookButton from '../../components/FacebookButton';
+import GoogleButton from '../../components/GoogleButton';
+import './signIn.css';
+import * as ROUTES from '../../constants/routes';
+
+const SignInForm = props => {
+  const {
+    form: { getFieldDecorator, validateFieldsAndScroll },
```
Comment (fares98) [inline src/containers/SignIn/index.js]: use validateFields insted of validateFieldsAndScroll

File: src/containers/SignIn/index.js [hunk 7]
```diff
@@ -1,8 +1,105 @@
...
+              ],
+            })(
+              <Input.Password
+                prefix={<Icon type="lock" className="signin__icon" />}
+                placeholder="Password"
+              />
+            )}
+          </Form.Item>
+
+          <Form.Item hasFeedback></Form.Item>
```
Comment (fares98) [inline src/containers/SignIn/index.js]: remove this line

File: src/containers/SignIn/index.js [hunk 8]
```diff
@@ -1,8 +1,105 @@
...
+          <Form.Item>
```
Comment (fares98) [inline src/containers/SignIn/index.js:69]: this element is not important

**Review by fares98 — CHANGES_REQUESTED** [review]

**Review by fares98 — APPROVED** [review]

**Review by AlaaTaima — APPROVED** [review]

**Review by Fatmasiam — APPROVED** [review]

**Review by Fatmasiam — APPROVED** [review]

**Review by yosefanajjar — APPROVED** [review]

**Pull request merged**